bento_beacon/authz/headers.py

```diff
@@ -0,0 +1,10 @@
+from flask import request, Request
```

Comment:
this file is just unused stuff, but we could leave it if you intend to do some non-bearer-token stuff in the future

Reply:
yeah, it's going to be in the katsu / gohan stuff